lib/airtable/entity/table.rb

```diff
@@ -0,0 +1,123 @@
+module Airtable
+  module Entity
+    # Airtable Table entity
+    class Table
+      PAGE_SIZE         = 100
+      DEFAULT_DIRECTION = 'asc'.freeze
+
+      def initialize(base, name)
+        @name = name
+        @base = base
+        @args = [@base, @name]
+      end
+
+      def select(options = {})
+        params = {}
+        update_default_params(params, options)
+        update_sort_options(params, options)
+        validate_params(params)
+        fetch_records(params.compact)
+      end
+
+      def find(id)
+        args = [@base, [@name, id].join('/')]
+        ::Airtable::Entity::Record.new(id).__fetch__(*args)
+      end
+
+      def create(fields)
+        ::Airtable::Entity::Record.new(nil, fields: fields).__create__(*@args)
```

Comment:
it would be clearer to spell out `*@args` as `[@base, @name]`

Reply:
Ok, i will try to change but possible rubocop will not like it.

Reply:
I see. If it does, lets use a more descriptive name, so it can't be confused with the method arguments. Right now someone might look at this and think that `*@args` is the arguments to `def create`.

Reply:
Done